src/pages/projects/forms/NetworkSelector.tsx

```diff
@@ -26,34 +28,77 @@ const NetworkSelector: FC<Props & SelectProps> = ({
   const getNetworkOptions = () => {
     const options = managedNetworks.map((network) => {
       return {
-        label: network.name,
+        label: (
+          <div className="label">
+            <span title={network.name} className="network-option u-truncate">
+              {network.name}
+            </span>
+            <span title={network.type} className="network-option u-truncate">
+              {network.type}
+            </span>
+            <span
+              title="network ACLs"
+              className="network-option u-truncate u-align--right"
+            >
+              {getNetworkAcls(network).length || "-"}
+            </span>
+          </div>
+        ),
         value: network.name,
+        text: (
+          <span>
+            {network.name}&nbsp;
+            <span className="u-text--muted">&#40;{network.type}&#41;</span>
+          </span>
+        ),
         disabled: false,
       };
     });
 
-    options.unshift({
-      label: options.length === 0 ? "No networks available" : "Select option",
-      value: "",
-      disabled: true,
-    });
+    if (options.length === 0) {
+      options.unshift({
+        label: <span>No networks available</span>,
+        value: "",
+        text: <></>,
+        disabled: true,
+      });
+    }
 
     if (hasNoneOption) {
-      options.push({ label: "No network", value: "none", disabled: false });
+      options.push({
+        label: <span>No network</span>,
+        value: "none",
+        text: <>none</>,
+        disabled: false,
+      });
     }
 
     return options;
   };
+
+  const getHeader = () => {
+    return (
+      <div className="header">
+        <span className="network-option u-no-margin--bottom">Name</span>
+        <span className="network-option u-no-margin--bottom">Type</span>
+        <span className="network-option u-no-margin--bottom">ACLs</span>
+      </div>
+    );
+  };
+
   return (
-    <Select
+    <CustomSelect
```

Comment:
There is a warning here, when opening this one:
```
Warning: Formik called `handleBlur`, but you forgot to pass an `id` or `name` attribute to your input:
```
I think to resolve it, we should remove the `onBlur` when calling the `NetworkSelector` in `NetworkDevice.tsx`.